framework/13-inch/common/audio.nix

```diff
@@ -0,0 +1,368 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.hardware.framework.laptop13.audioEnhancement;
+in
+{
+  options = {
+    hardware.framework.laptop13.audioEnhancement = {
+      enabled = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''
+          Create a new audio device called "Framework Speakers",
+          which applies sound tuning before sending the audio out to the speakers.
+          This option requires PipeWire and WirePlumber.
+
+          The filter chain includes the following:
+            - Pyschoacoustic bass enhancement
+            - Loudness compensation
+            - Equalizer
+            - Slight compression
+
+          This option has been optimised for the Framework Laptop 13 AMD 7040 series, but should work on all models.
+
+          Before applying, ensure the speakers are set to 100%,
+          because the volumes compound and the raw speaker device will be hidden by default.
+
+          In some cases, the added bass will vibrate the keyboard cable leading to a rattling sound,
+          a piece of foam can be used to mitigate this.
+        '';
+      };
+
+      hideRawDevice = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        description = ''
+          Hide the raw speaker device.
+          This option is enabled by default, because keeping the raw speaker device can lead to volume conflicts.
+        '';
+      };
+    };
+  };
+
+  config = lib.mkIf cfg.enabled (let
+    outputName = "alsa_output.pci-0000_c1_00.6.analog-stereo";
```

Comment:
I think this part needs to be configureable.
On framework 13 intel 13 Gen it's called:

```
 node.name = "alsa_output.pci-0000_00_1f.3.analog-stereo"
 node.nick = "92HD95 Analog"
 media.class = "Audio/Sink"
```

Reply:
I changed the configuration, but the result was no I now have no audio: https://github.com/Mic92/nixos-hardware/commit/23ef53a7c46b4f3b155ce74e1e6bd7886054f431#diff-2dbd4c2fc043321e7d7fd56ee926f400d2e48223add74cfd1aa2a867bc499e74R174
Toggling the option and restarting the pipewire systemd service brings audio back.
I did have my audio on 100% before activating.

Reply:
You might have to re-select the default output device the first time you use this. I'll add that to the description (and the configurable device name) soon

Reply:
Ah. Yes. Now I got it working!